source/administration/extended-support-release.rst

```diff
@@ -44,7 +44,9 @@ What are the current supported Extended Support Release versions?
 +-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
 | Version     | Type           | Release Date     | End of Support   | Latest Dot Release Download link                                                           | Upgrade Notes                                   |
 +=============+================+==================+==================+============================================================================================+=================================================+
-| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.4 <https://releases.mattermost.com/5.25.4/mattermost-5.25.4-linux-amd64.tar.gz>`_    |                                                 |
+| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.5 <https://releases.mattermost.com/5.25.5/mattermost-5.25.5-linux-amd64.tar.gz>`_    | SAML Setting "Use Improved SAML                 |
+|             |                |                  |                  |                                                                                            | Library (Beta)" was disabled.                   |
+|             |                |                  |                  |                                                                                            | Please use ``xmlsec1``.                         |
```

Comment:
Propose

```
Please use ``xmlsec1`` utility <https://docs.mattermost.com/deployment/sso-saml-before-you-begin.html>`_.
```

with a link to the instructions.

Reply:
I missed this comment in the urgency to merge the updates, sorry about that.